code/game/objects/items/robot/robot_items.dm

```diff
@@ -946,3 +946,26 @@
 	. = ..()
 	if(istype(A, /obj/item/aiModule) && !stored) //If an admin wants a borg to upload laws, who am I to stop them? Otherwise, we can hint that it fails
 		to_chat(user, "<span class='warning'>This circuit board doesn't seem to have standard robot apparatus pin holes. You're unable to pick it up.</span>")
+		
+////////////////////
+//versatile service holder//
+////////////////////
+
+/obj/item/borg/apparatus/beaker/service
+	name = "versatile service grasper"
+	desc = "Specially designed for carrying glasses, food and seeds. Alt-Z or right-click to drop the stored object."
+	storable = list(/obj/item/reagent_containers/food,
+	/obj/item/seeds,
+	/obj/item/storage/fancy/donut_box,
+	/obj/item/storage/fancy/egg_box,
+	/obj/item/clothing/mask/cigarette,
+	/obj/item/storage/fancy/cigarettes,
+	/obj/item/reagent_containers/glass/beaker,
+	/obj/item/reagent_containers/glass/bottle,
+	/obj/structure/mopbucket
```

Comment:
This seems more janitor related than service related, and the mopbucket is something that should be dragged around rather than held if I am thinking of the right item.

Reply:
Right. Fixed.